### ISSUE TITLE
feat: v3 to v4 wizzard

### ISSUE THREAD

**Issue by arashsheyda** [nuxt/devtools#759]
### 🆒 Your use case

should we add the option to migrate a nuxt 3 project to nuxt 4?

### 🆕 The solution you'd like

using the codemod script in devtools wizard to migrate it
and eventually show an alert to user when Nuxt 4 is fully out?

**Comment by antfu** [comment]
I like the idea but I am also a bit worried about the UX. It would be great to do one-click migration in the UI, but I am afraid that for most of the cases, it would still requires manually changes, which we couldn't handle in the UI. In that case I feel it might be better to ask them to run the command manually where they would also have more controls. For us, we could provide a tip message for people to read the migration guide on the docs site. WDYT?

**Comment by arashsheyda** [comment]
yes that sounds good! should we implement it when Nuxt 4 is fully out or...

**Comment by antfu** [comment]
Yeah, we could wait, I think. The main traffic of the migration would be the release announcement anyway, our tip would only be a small supplement.
